README.md

```diff
@@ -4,56 +4,130 @@
 [![Build Status](https://travis-ci.org/fthomas/singleton-ops.svg?branch=master)](https://travis-ci.org/fthomas/singleton-ops)
 [![codecov](https://codecov.io/gh/fthomas/singleton-ops/branch/master/graph/badge.svg)](https://codecov.io/gh/fthomas/singleton-ops)
 
-This is a proof of concept library that provides operations for primitive
-and `String` singleton types. It lifts value-level operations of these types
-to the type-level and allows for example to add `Double`s at the type-level.
+This library provides type-level operations for [Typelevel Scala][typelevel-scala] with [SIP-23][sip-23].
+
+####Supported types:
+* `Int` 
+* `Long`
+* `Double`
+* `String`
+* `Boolean`
+* `Nat` (from [Shapeless][shapeless])
```

Comment:
Next version :) 
It's not tough to add them, but supporting the different implicit operations is very cumbersome as an implementation, so I had decided they will be implemented once we see that the current code structure holds well.